x-pack/plugins/session_view/public/components/process_tree/helpers.ts

```diff
@@ -5,8 +5,22 @@
  * 2.0.
  */
 import { Process, ProcessEvent, ProcessMap } from '../../../common/types/process_tree';
+import { UpdateAlertStatus } from '../../types';
 import { ProcessImpl } from './hooks';
 
+// if given event is an alert, and it exist in updatedAlertsStatus, update the alert's status
+// with the updated status value in updatedAlertsStatus Map
+export const updateAlertEventStatus = (
+  events: ProcessEvent[],
+  updatedAlertsStatus: UpdateAlertStatus
+) => {
+  events.forEach((event) => {
+    if (event.kibana?.alert.uuid && updatedAlertsStatus[event.kibana.alert.uuid]) {
+      event.kibana.alert.workflow_status = updatedAlertsStatus[event.kibana.alert.uuid].status;
```

Comment:
With Kibana, I would never assume you'll actually get the data so I would do `updatedAlertStatus[event.kibana?.alert?.uuid]`

Reply:
Thanks for letting me know! There are other places in session_view that will require similar fixes. I will fix this one here and create a tech debt ticket to address the rest in a separate PR.

Reply:
Great, thanks!